gc/base/standard/Scavenger.cpp

```diff
@@ -5932,6 +5943,33 @@ MM_Scavenger::completeConcurrentCycle(MM_EnvironmentBase *env, MM_MemorySubSpace
 	}
 }
 
+void
+MM_Scavenger::payAllocationTax(MM_EnvironmentBase *env, MM_MemorySubSpace *subspace, MM_MemorySubSpace *baseSubSpace, MM_AllocateDescription *allocDescription)
+{
+	if (isCurrentPhaseConcurrent()) {
+		uintptr_t freeMemory = _activeSubSpace->getMemorySubSpaceAllocate()->getApproximateActiveFreeMemorySize();
+		uintptr_t totalMemory = _activeSubSpace->getMemorySubSpaceAllocate()->getActiveMemorySize();
+
+		/* Progress of how much Survivor/Allocate area has been consumed since the start this cycle relative.
+		 * (Although fairly small, consider excluding the amount consumed by the first STW increment.
```

Comment:
extra bracket can be removed
```suggestion
		 * Although fairly small, consider excluding the amount consumed by the first STW increment.
```